src/shogun2-util/src/main/java/de/terrestris/shogun2/util/interceptor/OgcMessage.java

```diff
@@ -0,0 +1,382 @@
+	 * @return
+	 */
+	public boolean isWmsGetStyles() {
+		return (this.isWms() &&
+				this.getOperation().equalsIgnoreCase(OgcNaming.OPERATION_GET_STYLES)) ?
+				true : false;
+	}
+
+	/**
+	 *
+	 * @return
+	 */
+	public boolean isWfsGetCapabilities() {
+		return (this.isWfs() &&
+				this.getOperation().equalsIgnoreCase(OgcNaming.OPERATION_GET_CAPABILITIES)) ?
+				true : false;
+	}
+
+	/**
+	 *
+	 * @return
+	 */
+	public boolean isWfsDescribeFeatureType() {
+		return (this.isWfs() &&
+				this.getOperation().equalsIgnoreCase(OgcNaming.OPERATION_DESCRIBE_FEATURE_TYPE)) ?
+				true : false;
+	}
+
+	/**
+	 *
+	 * @return
+	 */
+	public boolean isWfsGetFeature() {
+		return (this.isWfs() &&
+				this.getOperation().equalsIgnoreCase(OgcNaming.OPERATION_GET_FEATURE)) ?
+				true : false;
+	}
+
+	/**
+	 *
+	 * @return
+	 */
+	public boolean isWfsLockFeature() {
+		return (this.isWfs() &&
+				this.getOperation().equalsIgnoreCase(OgcNaming.OPERATION_LOCK_FEATURE)) ?
+				true : false;
+	}
+
+	/**
+	 *
+	 * @return
+	 */
+	public boolean isWfsTransaction() {
+		return (this.isWfs() &&
+				this.getOperation().equalsIgnoreCase(OgcNaming.OPERATION_TRANSACTION)) ?
+				true : false;
+	}
+
+	/**
+	 *
+	 * @return
+	 */
+	public boolean isWcsGetCapabilities() {
+		return (this.isWcs() &&
+				this.getOperation().equalsIgnoreCase(OgcNaming.OPERATION_GET_CAPABILITIES)) ?
+				true : false;
+	}
+
+	/**
+	 *
+	 * @return
+	 */
+	public boolean isWcsDescribeCoverage() {
+		return (this.isWcs() &&
+				this.getOperation().equalsIgnoreCase(OgcNaming.OPERATION_DESCRIBE_COVERAGE)) ?
+				true : false;
+	}
+
+	/**
+	 *
+	 * @return
+	 */
+	public boolean isWcsGetCoverage() {
+		return (this.isWcs() &&
+				this.getOperation().equalsIgnoreCase(OgcNaming.OPERATION_GET_COVERAGE)) ?
+				true : false;
+	}
+
+	/**
+	 *
+	 * @return
+	 */
+	public boolean isRequestAllowed() {
+		return this.getRequestRule().equalsIgnoreCase("ALLOW");
+	}
+
+	/**
+	 *
+	 * @return
+	 */
+	public boolean isResponseAllowed() {
+		return this.getResponseRule().equalsIgnoreCase("ALLOW");
+	}
+
+	/**
+	 *
+	 * @return
+	 */
+	public boolean isRequestDenied() {
+		return this.getRequestRule().equalsIgnoreCase("DENY");
+	}
+
+	/**
+	 *
+	 * @return
+	 */
+	public boolean isResponseDenied() {
+		return this.getResponseRule().equalsIgnoreCase("DENY");
+	}
+
+	/**
+	 *
+	 * @return
+	 */
+	public boolean isRequestModified() {
+		return this.getRequestRule().equalsIgnoreCase("MODIFY");
+	}
+
+	/**
+	 *
+	 * @return
+	 */
+	public boolean isResponseModified() {
+		return this.getResponseRule().equalsIgnoreCase("MODIFY");
+	}
+
+	/**
+	 *
+	 */
+	@Override
+	public boolean equals(Object obj) {
+		if (!(obj instanceof OgcMessage))
+			return false;
+		OgcMessage other = (OgcMessage) obj;
+
+		return new EqualsBuilder()
+				.appendSuper(super.equals(other))
+				.append(getService(), other.getService())
+				.append(getOperation(), other.getOperation())
+				.append(getEndPoint(), other.getEndPoint())
```

Comment:
As we have discussed, we sould have the response/resquest rule here also. I am unsure wrt to the use of `.appendSuper(super.equals(other))`, I'd favor a short note of @buehner here.